blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,88 @@
+1. You will also need a work or school Microsoft Account (MSA). Personal or guest MSAs cannot be used with the Dev Box portal at this time.
+1. Your machine will need to be managed with [Microsoft Intune](https://www.microsoft.com/security/business/Microsoft-Intune).
+
+I followed Microsoft's documentation to create a dev box using Microsoft Dev Box. I found the documentation to be straightforward to follow, and there are also several tutorial videos online from developers if you prefer to see someone walk through the steps live. You'll need to complete two steps to get your dev box up and running: [Setting Up Your Dev Box Service](https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service) and [Creating a Dev Box](https://learn.microsoft.com/azure/dev-box/quickstart-create-dev-box). These steps should take about an hour to complete.
+
+Once your dev box is running, you will be able to use the virtual machine as if you are on a Windows computer.
+
+Same as with Parallels above, you can follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows to get set up.
+
+Unlike Parallels, dev box runs in a cloud and does not have access to your local files, so you will need to redownload any source code that you need to get started developing. When you are finished with your development session and have shut down the virtual machine in the Dev Box portal, your machine's state will be saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-MicrosoftDevBox.jpg)
+
+## Takeaways about Microsoft Dev Box
+
+Microsoft Dev Box is a great tool when your team is ready for a larger level of investment in the Windows platform. Its feature offerings of cloud-based infrastructure, pre-configured dev boxes, and Intune secured workstations are great for enterprise-scale development.
+
+Microsoft Dev Box requires several [admin and configuration steps](https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service) to get started, so if you're looking to quickly experiment with RNW within a VM, Parallels or another virtual machine offering may be a good option to get started with. Then once you're ready to scale your Windows VM usage across many developers to build large production experiences, Microsoft Dev Box could become a better option.
```

Comment:
Here you use the `RNW` and `VM` acronyms, which it feels like you're avoiding in the rest of the article.